Userland/Utilities/image.cpp

```diff
@@ -87,7 +87,13 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         bytes = TRY(Gfx::PNGWriter::encode(*frame, { .icc_data = icc_data }));
     } else if (out_path.ends_with(".ppm"sv, CaseSensitivity::CaseInsensitive)) {
         auto const format = ppm_ascii ? Gfx::PortableFormatWriter::Options::Format::ASCII : Gfx::PortableFormatWriter::Options::Format::Raw;
-        bytes = TRY(Gfx::PortableFormatWriter::encode(*frame, { .format = format }));
+
+        auto output_stream = TRY(Core::File::open(out_path, Core::File::OpenMode::Write));
```

Comment:
Let's just move the creation of `output_stream` and `buffered_stream` before the row of if-blocks and remove the duplicates below.